core/src/main/python/wlsdeploy/tool/util/archive_helper.py

```diff
@@ -401,6 +401,20 @@ def extract_database_wallet(self, wallet_name=WLSDeployArchive.DEFAULT_RCU_WALLE
         self.__logger.exiting(class_name=self.__class_name, method_name=_method_name, result=resulting_wallet_path)
         return resulting_wallet_path
 
+    def extract_all_database_wallets(self):
+        archive_entries = self.get_archive_entries()
+        wallet_names = sets.Set()
+        for entry in archive_entries:
+            if entry.startswith(WLSDeployArchive.ARCHIVE_DB_WALLETS_DIR):
+                if os.path.isdir(entry):
+                    name = wallet_names.add(os.path.basename(entry))
```

Comment:
Set().add() does not return anything, so name will be void for this code path.  I don't think that is what you intended.

Reply:
That's unintentional, fixed